runtime/polkadot/src/governance/tracks.rs

```diff
@@ -0,0 +1,320 @@
+			min_approval: APP_REFERENDUM_CANCELLER,
+			min_support: SUP_REFERENDUM_CANCELLER,
+		},
+	),
+	(
+		21,
+		pallet_referenda::TrackInfo {
+			name: "referendum_killer",
+			max_deciding: 1_000,
+			decision_deposit: 50 * GRAND,
+			prepare_period: 2 * HOURS,
+			decision_period: 28 * DAYS,
+			confirm_period: 3 * HOURS,
+			min_enactment_period: 10 * MINUTES,
+			min_approval: APP_REFERENDUM_KILLER,
+			min_support: SUP_REFERENDUM_KILLER,
+		},
+	),
+	(
+		30,
+		pallet_referenda::TrackInfo {
+			name: "small_tipper",
+			max_deciding: 200,
+			decision_deposit: 1 * DOLLARS,
+			prepare_period: 1 * MINUTES,
+			decision_period: 7 * DAYS,
+			confirm_period: 10 * MINUTES,
+			min_enactment_period: 1 * MINUTES,
+			min_approval: APP_SMALL_TIPPER,
+			min_support: SUP_SMALL_TIPPER,
+		},
+	),
+	(
+		31,
+		pallet_referenda::TrackInfo {
+			name: "big_tipper",
+			max_deciding: 100,
+			decision_deposit: 10 * DOLLARS,
+			prepare_period: 10 * MINUTES,
+			decision_period: 7 * DAYS,
+			confirm_period: 1 * HOURS,
+			min_enactment_period: 10 * MINUTES,
+			min_approval: APP_BIG_TIPPER,
+			min_support: SUP_BIG_TIPPER,
+		},
+	),
+	(
+		32,
+		pallet_referenda::TrackInfo {
+			name: "small_spender",
+			max_deciding: 50,
+			decision_deposit: 100 * DOLLARS,
+			prepare_period: 4 * HOURS,
+			decision_period: 28 * DAYS,
+			confirm_period: 12 * HOURS,
+			min_enactment_period: 24 * HOURS,
+			min_approval: APP_SMALL_SPENDER,
+			min_support: SUP_SMALL_SPENDER,
+		},
+	),
+	(
+		33,
+		pallet_referenda::TrackInfo {
+			name: "medium_spender",
+			max_deciding: 50,
+			decision_deposit: 200 * DOLLARS,
+			prepare_period: 4 * HOURS,
+			decision_period: 28 * DAYS,
+			confirm_period: 24 * HOURS,
+			min_enactment_period: 24 * HOURS,
+			min_approval: APP_MEDIUM_SPENDER,
+			min_support: SUP_MEDIUM_SPENDER,
+		},
+	),
+	(
+		34,
+		pallet_referenda::TrackInfo {
+			name: "big_spender",
+			max_deciding: 50,
+			decision_deposit: 400 * DOLLARS,
+			prepare_period: 4 * HOURS,
+			decision_period: 28 * DAYS,
+			confirm_period: 48 * HOURS,
+			min_enactment_period: 24 * HOURS,
+			min_approval: APP_BIG_SPENDER,
+			min_support: SUP_BIG_SPENDER,
+		},
+	),
+];
+
+pub struct TracksInfo;
+impl pallet_referenda::TracksInfo<Balance, BlockNumber> for TracksInfo {
+	type Id = u16;
+	type RuntimeOrigin = <RuntimeOrigin as frame_support::traits::OriginTrait>::PalletsOrigin;
+	fn tracks() -> &'static [(Self::Id, pallet_referenda::TrackInfo<Balance, BlockNumber>)] {
+		&TRACKS_DATA[..]
+	}
+	fn track_for(id: &Self::RuntimeOrigin) -> Result<Self::Id, ()> {
+		if let Ok(system_origin) = frame_system::RawOrigin::try_from(id.clone()) {
+			match system_origin {
+				frame_system::RawOrigin::Root => Ok(0),
+				_ => Err(()),
+			}
+		} else if let Ok(custom_origin) = origins::Origin::try_from(id.clone()) {
+			match custom_origin {
+				origins::Origin::WhitelistedCaller => Ok(1),
+				// General admin
+				origins::Origin::StakingAdmin => Ok(10),
```

Comment:
worth adding a comment that 2-9 are fellowship tracks

Reply:
The fellowship-referenda pallet configuration (and its track indices) and the general-referenda pallet configuration (and its track indices) are entirely independent.